tests/mock_crust/utils.rs

```diff
@@ -261,13 +261,34 @@ impl TestClient {
         endpoint: Option<Endpoint>,
         full_id: FullId,
     ) -> Self {
+        let duration = Duration::from_secs(CLIENT_MSG_EXPIRY_DUR_SECS);
+        Self::new_with_all(network, bootstrap_config, endpoint, full_id, duration)
+    }
+
+    pub fn new_with_expire_duration(
+        network: &Network<PublicId>,
+        bootstrap_config: Option<BootstrapConfig>,
+        endpoint: Option<Endpoint>,
+        duration: Duration,
+    ) -> Self {
+        let full_id = FullId::new();
+        Self::new_with_all(network, bootstrap_config, endpoint, full_id, duration)
+    }
+
+    fn new_with_all(
```

Comment:
better to call this `new_impl` if other ctors are calling it. `new_with_all` is difficult to follow while reading.